autoload/copilot_chat/api.vim

```diff
@@ -2,15 +2,24 @@ scriptencoding utf-8
 
 let s:curl_output = []
 
-function! copilot_chat#api#async_request(message) abort
+function! copilot_chat#api#async_request(message, file_list=[]) abort
```

Comment:
makes me incredibly sad that vim-jp doesnt actually support optional arguments.. project has been stagnant for 2+ years and I assume its vim9script era feature. Extra motivation get my ruby linter going..

```
autoload/copilot_chat/api.vim:5:61: unexpected token: = (see vim-jp/vim-vimlparser)
```

Reply:
My linter isnt ready here so im probably looking at merging this change and breaking the linter. Ill own the manual review of people using the linter + make sure theres an issue created within the repo